crates/astria-conductor/src/executor/mod.rs

```diff
@@ -417,15 +417,15 @@ impl Executor {
     /// returns the previously-computed execution block hash.
     #[instrument(skip(self), fields(sequencer_block_hash = ?block.block_hash, sequencer_block_height = block.header.height.value()))]
     async fn execute_block(&mut self, block: SequencerBlockSubset) -> Result<Block> {
-        let executable_block_height = self.get_executable_block_height()?;
-        if u64::from(executable_block_height) != block.header.height.value() {
-            error!(
-                sequencer_block_height = block.header.height.value(),
-                executable_block_height = executable_block_height,
-                "block received out of order;"
-            );
-            return Err(eyre!("block received out of order"));
```

Comment:
this is doubly reported. Actual errors should be handled at the top most level, not deep in the stack.